src/urh/dev/PCAPNG.py

```diff
@@ -0,0 +1,115 @@
+import os
+import struct
+import math
+
+from urh.util.Logger import logger
+
+
+# Refer to PCAPNG spec
+# https://www.ietf.org/staging/draft-tuexen-opsawg-pcapng-02.html
+
+class PCAPNG(object):
+    def __init__(self):
+        self.__messages = []
+        self.__messages_timestamps = []
+        self.__filename = ""
+
+    def __build_pcapng_shb(self, shb_userappl: str = "", shb_hardware: str = "") -> bytes:
+        BLOCKTYPE = 0x0A0D0D0A
```

Comment:
method variables should not be uppercased.

Reply:
Those are constants. Unless you have another convention in your project, could be a good idea to use uppercases to distinguish constants.